### ISSUE TITLE
Is it possible to get xref (cross reference) to a member of a custom struct?

### ISSUE THREAD

**Issue by ddbb2017** [NationalSecurityAgency/ghidra#474]
Say, if I created my own custom data type (i.e. `struct`):

![Capture1](https://user-images.githubusercontent.com/33359889/56455181-2dc4a780-6310-11e9-8baa-b4cff8ce2bc9.PNG)

I know that I can currently find cross-references to functions and global variables, but I'm curious, if it is possible to find cross-references to one of the members of my custom data type (`cpxThumb` in this case) as it is being used in the decompiled code?

I'm asking for an option that would work throughout an entire program, as I can obviously do it from within a single function by middle-mouse-clicking the variable name.

Instances like this:

![Capture2](https://user-images.githubusercontent.com/33359889/56455177-2ac9b700-6310-11e9-8680-c883329fc3e1.PNG)

**Comment by jrmuizel** [comment]
Yes. Right clicking on the type in the "Data Type Manager" gives a context menu with a "Find uses of field..." entry. I believe that should do what you're looking for.

**Comment by ddbb2017** [comment]
@jrmuizel oh, awesome. That's exactly what I was looking for. Thanks!

(I'll re-tag it into a suggestion now.) It would be really helpful if there was a context menu item in the "Structure editor" window that would do the same. Here's an example:

![Untitled-1](https://user-images.githubusercontent.com/33359889/56462050-37322c00-6371-11e9-9792-6c7cac00b703.png)

Also, as a second suggestion. The "Structure Editor" window should allow multiple instances of self to be opened. For instance, I may be working on editing one struct and I might want to open the editor for another struct at the same time. At the current moment if I try to do this, the old instance closes up.
